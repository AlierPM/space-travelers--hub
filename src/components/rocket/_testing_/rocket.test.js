/* eslint-disable */
import renderer from 'react-test-renderer';
import Rocket from '../rocket';

const rocket1 = {
  id: 1,
  rocket_name: 'First Space X Rocket',
  describe: 'Details about first space1 rocket',
  image: 'https://farm1.staticflickr.com/929/28787338307_3453a11a77_b.jpg',
  reserve: true,
  cancel: true,
  reserved: false,
};

it('Check for the first space x rocket', () => {
  const { id, rocket_name, describe, image, reserve, cancel, reserved } =
    rocket1;
  const firstRocket = renderer
    .create(
      <Rocket
        id={id}
        rocket_name={rocket_name}
        describe={describe}
        image={image}
        reserve={reserve}
        cancel={cancel}
        reserved={reserved}
      />,
    )
    .toJSON();
  expect(firstRocket).toMatchSnapshot();
});

const rocket2 = {
  id: 2,
  rocket_name: 'Second Space X',
  describe: 'More details About Second Space X rocket',
  image: 'https://farm1.staticflickr.com/929/28787338307_3453a11a77_b.jpg',
  reserve: false,
  cancel: true,
  reserved: false,
};
it('Check for the Second Space X rocket', () => {
  const {id, rocket_name, describe, image, reserve, cancel, reserved} = rocket2
  const secondRocket = renderer.create(<Rocket
    id = {id}
    rocket_name= {rocket_name}
    describe={describe}
    image={image}
    reserve={reserve}
    cancel={cancel}
    reserved ={reserved}
    />).toJSON();
    expect(secondRocket).toMatchSnapshot();
});

const rocket3 = {
  id: 3,
  rocket_name: 'Third Space X',
  describe: 'More details on the Third space x rocket',
  image: '	https://farm5.staticflickr.com/4599/38583829295_581f34dd84_b.jpg',
  reserve: false,
  cancel: true,
  reserved: true
};
it('Check for the Third Space X Rocket', () => {
  const {id, rocket_name, describe, image, reserve, cancel, reserved} = rocket3;
  const thirdRocket = renderer.create(<Rocket 
  id={id}
  rocket_name= {rocket_name}
  describe={describe}
  image={image}
  reserve ={reserve}
  cancel={cancel}
  reserved ={reserved}
  />)
  expect(thirdRocket).toMatchSnapshot();
});