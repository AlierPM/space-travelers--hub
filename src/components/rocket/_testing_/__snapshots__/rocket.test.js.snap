// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check for second Space X hub 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="https://farm1.staticflickr.com/929/28787338307_3453a11a77_b.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        Space X 2
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Check for the Second Space X rocket 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="https://farm1.staticflickr.com/929/28787338307_3453a11a77_b.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        Second Space X
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Check for the Third Space X Rocket 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="	https://farm5.staticflickr.com/4599/38583829295_581f34dd84_b.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        Third Space X
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Check for the Third Space X hub 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="	https://farm5.staticflickr.com/4599/38583829295_581f34dd84_b.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        Space X 3
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Check for the first Space X haub 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="https://imgur.com/DaCfMsj.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        Space X 1
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;

exports[`Check for the first space x rocket 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <img
      alt="Rocket"
      className="rocket col-4"
      src="https://farm1.staticflickr.com/929/28787338307_3453a11a77_b.jpg"
    />
    <div
      className="col-8"
    >
      <h3
        className="col-12"
      >
        First Space X Rocket
      </h3>
      <p
        className="col-12 my-4"
      >
        <span
          className="reserve-text"
        >
           
           
        </span>
      </p>
       
      <button
        className="reserve-btn p-3 col-4"
        onClick={[Function]}
        type="button"
      >
         
        Reserve Rocket
      </button>
    </div>
  </div>
</div>
`;
